### DISCUSSION TITLE
How to style prefix of q-input with different color

### DISCUSSION THREAD

**Discussion by 549393092** [quasarframework/quasar#15705] [General - Components / Directives / etc] [answered]
Dear Quasar team,

I can override the color for the prefix, but only with one color at one time. How to with different colors?
https://codepen.io/Stanley-549393092/pen/zYmrKVa

Regards

**Comment by 549393092** [comment, marked as answer]
It's related to css syntax.

**Reply by suleimanas** [reply]
Could you elaborate more please
